### ISSUE TITLE
A new version would be nice

### ISSUE THREAD

**Issue by fabio42** [76creates/stickers#13]
Hello I'm trying your table package (and it looks great)!

The current latest tag version v1.3.0 points to a relatively old version that don't contains the table package. From what I can see there have been a lots of changes since that version. The consequence is that trying to import it in another project result with the following error:
```
github.com/76creates/stickers/table: module github.com/76creates/stickers@latest found (v1.3.0), but does not contain package github.com/76creates/stickers/table
```

Maybe there is  a good reason that I don't see for not having released a new version yet, but looking at the provided example,
this almost force to fork your code locally rather than just importing it.

**Comment by fabio42** [comment]
I learn little bit more of go and been able to import the library by doing:
```
go get github.com/76creates/stickers@v1.3.1-0.20230410064447-c0cf398aec57

**Comment by 76creates** [comment]
Not a lot of development was done recently, I couldn't find time for stickers, Ill try to improve dev experience in following days/weeks.

**Comment by BrownNPC** [comment]
Years*

**Comment by 76creates** [comment]
> Years*

Phew, ok, now this is motivation.
 _Adding "stickers maintenance" to the callendar for this week..._

**Comment by 76creates** [comment]
Released `1.4.0` looking forward to make some update for this one. 
Thanks @BrownNPC for sparking [inat](https://www.opendemocracy.net/en/article_2233jsp/) within me, hope it holds long time.
